### ISSUE TITLE
i'm not getting output. facing d below error..pls try to solve my issue

### ISSUE THREAD

**Issue by SusmithaRachamalla** [funador/react-image-upload#1]
ReferenceError: basicdemo is not defined
    at Object.<anonymous> (C:\Users\Susmitha\Downloads\react-image-upload-master\server\server.js:11:15)
    at Module._compile (module.js:653:30)
    at Object.Module._extensions..js (module.js:664:10)
    at Module.load (module.js:566:32)
    at tryModuleLoad (module.js:506:12)
    at Function.Module._load (module.js:498:3)
    at Function.Module.runMain (module.js:694:10)
    at startup (bootstrap_node.js:204:16)
    at bootstrap_node.js:625:3
[nodemon] app crashed - waiting for file changes before starting...

**Comment by funador** [comment]
That is an error for trying to access a variable (basicdemo) that does not exist.  Pretty sure it has nothing to do with this repo and is code you have added.

**Comment by funador** [comment]
Going to close this as I am pretty sure it is related to code that was added. Feel free to open another issue if that is not the case. Cheers.
